behaviors/TranslatableModel.php

```diff
@@ -33,7 +33,7 @@ public function __construct($model)
 
     /**
      * Applies a translatable index to a basic query. This scope will join the index
-     * table and cannot be executed more than once.
+     * table and can be executed neither more than once, nor with scopeTransOrder.
```

Comment:
@mjauvin what does this mean?

Reply:
Indeed... @aurelien-roy can you fix this comment so it makes sense?